policyengine_us/tests/policy/baseline/gov/states/pa/tax/income/pa_use_tax.yaml

```diff
@@ -18,10 +18,11 @@
     pa_use_tax: 7
 
 
-- name: PA income tax before forgiveness is 3.07%.
+- name: PA income tax before forgiveness is 3.07%, rest of PA.
```

Comment:
While you're here could you fix these test names? It's about use tax not forgiveness